superdesk/internal_destinations.py

```diff
@@ -73,7 +73,7 @@ def handle_item_published(sender, item, **extra):
         if dest.get('macro'):
             macro = macros_service.get_macro_by_name(dest['macro'])
             try:
-                macro['callback'](new_item)
+                macro['callback'](new_item, dest_desk_id=dest.get('desk'))
```

Comment:
you could also add stage to be consistent

Reply:
oh yes good catch, I've just overlooked it, adding it now.